### PR TITLE
feature: Add FreeBSD NFS Client/Server application support

### PR DESCRIPTION
**Pull request by VVelox**
DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [X] Have you signed the [Contributors agreement](http://docs.librenms.org/General/Contributing/) - please do NOT submit a pull request unless you have (signing the agreement in the same pull request is fine). Your commit message for signing the agreement must appear as per the docs.
- [X] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`

This depends on https://github.com/librenms/librenms-agent/pull/90 .

### CODE REVIEW

**Comment by VVelox** [comment]
Removed unneeded require line.

File: includes/polling/applications/fbsd-nfs-client.inc.php [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+$app_id = $app['app_id'];
+
+use LibreNMS\RRD\RrdDefinition;
+
+$options      = '-O qv';
+$mib          = 'NET-SNMP-EXTEND-MIB';
+$oid          = 'nsExtendOutLine.13.102.98.115.100.110.102.115.99.108.105.101.110.116';
+$nfsclient = snmp_walk($device, $oid, $options, $mib);
+
+list( $Getattr, $Setattr, $Lookup, $Readlink, $Read, $Write, $Create, $Remove, $Rename, $Link, $Symlink, $Mkdir, $Rmdir,
```
Comment (laf) [inline includes/polling/applications/fbsd-nfs-client.inc.php]: Can you remove the space after `list(`

File: includes/polling/applications/fbsd-nfs-server.inc.php [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+use LibreNMS\RRD\RrdDefinition;
+
+$name = 'fbsd-nfs-server';
+$app_id = $app['app_id'];
+$options      = '-O qv';
+$mib          = 'NET-SNMP-EXTEND-MIB';
+$oid          = 'nsExtendOutLine.13.102.98.115.100.110.102.115.115.101.114.118.101.114';
+$nfsserver = snmp_walk($device, $oid, $options, $mib);
+
+list( $Getattr, $Setattr, $Lookup, $Readlink, $Read, $Write, $Create, $Remove, $Rename, $Link, $Symlink,
```
Comment (laf) [inline includes/polling/applications/fbsd-nfs-server.inc.php]: Space on `list( ` again.

Comment (laf) [inline includes/polling/applications/fbsd-nfs-server.inc.php]: One at the end.

**Comment by VVelox** [comment]
Sweet. All changes made now.

**Comment by laf** [comment]
Can you change all variables to lowercase please.

**Pull request merged**